examples/gcoap/gcoap_cli.c

```diff
@@ -32,11 +32,14 @@
 static void _resp_handler(unsigned req_state, coap_pkt_t* pdu,
                           sock_udp_ep_t *remote);
 static ssize_t _stats_handler(coap_pkt_t* pdu, uint8_t *buf, size_t len);
+static ssize_t _riot_board_handler(coap_pkt_t* pdu, uint8_t *buf, size_t len);
 
 /* CoAP resources */
 static const coap_resource_t _resources[] = {
-    { "/cli/stats", COAP_GET, _stats_handler },
+    { "/cli/stats", COAP_GET | COAP_PUT | COAP_POST, _stats_handler },
```

Comment:
The function simply replaces the resource value, so I think this entry should include only COAP_PUT.

Reply:
You are right